src/helper/index.tsx

```diff
@@ -26,6 +26,8 @@ export interface Tokens {
   chainId?: NetworkChainId;
   bridgeTo?: Array<NetworkChainId>;
 }
+export const ledgerChainIds: { [key: number]: NetworkChainId } = { 60: 'injective-1', 118: 'Oraichain' };
```

Comment:
any chance if we can config this somewhere?

Reply:
We can config them in the sdk if that is necessary